holmes.go

```diff
@@ -521,3 +529,20 @@ func (h *Holmes) EnableDump(curCPU int) (err error) {
 	}
 	return nil
 }
+
+func (h *Holmes) Set(opts ...Option) error {
+	h.opts.L.Lock()
+	defer h.opts.L.Unlock()
+	for _, opt := range opts {
+		if err := opt.apply(h.opts); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (h *Holmes) GetOpts() options {
```

Comment:
what does this used for?

Reply:
It's an outdated method, it won't be used, so I deleted it.